act/registry.go

```diff
@@ -229,41 +233,68 @@ func (r *Registry) Run(
 	// Prepend the logger to the parameters.
 	params = append([]sdkAct.Parameter{WithLogger(r.logger)}, params...)
 
+	timeout := r.defaultActionTimeout
+	if action.Timeout > 0 {
+		timeout = time.Duration(action.Timeout) * time.Second
+	}
+	var ctx context.Context
+	var cancel context.CancelFunc
+	// if timeout is zero, then the context should not have timeout
+	if timeout > 0 {
+		ctx, cancel = context.WithTimeout(context.Background(), timeout)
+	} else {
+		ctx, cancel = context.WithCancel(context.Background())
+
```

Comment:
Empty line:
```suggestion

```

Reply:
good catch. forgot to ran `golangci-lint`. It is fixed now.